extras/packaging-scripts/package-lambda.sh

```diff
@@ -17,7 +17,7 @@
 #           --src_dir ~/Security-Reference-Architecture/solutions/cloudtrail/cloudtrail-org/code/src
 ###########################################################################################
 
-usage="$(basename "$0") [-h] <--file_name s> [--bucket s] <--src_folder s> ---script to package lambda zip and upload to s3
+usage="$(basename \"$0\") [-h] <--file_name s> [--bucket s] <--src_folder s> ---script to package lambda zip and upload to s3
```

Comment:
Can you provide details on the error you got with the original file? 

Removing the quotes around the $0 gets me the same result but I don't get the shellcheck SC2086 error when I include the double quotes around the $0.